data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+	on offer
+		event "looking for a gun" 30
+
+event "looking for a gun"
+
+mission "Unfettered: Non Lethal Weapons Hint"
+	name `Learn more about the Pug.`
+	description `Go to <destination> to see if you can ask Admiral Danforth or the Oathkeepers about what kind of weapons the Pug used, which may be of interest to He Yi.`
+	minor
+	source
+		attributes "unfettered"
+	destination "Farpoint"
+	to offer
+		has "event: looking for a gun"
+		has "Unfettered: Close Quarter: active"
+	on offer
+		conversation
+			`As you land on this Unfettered planet, you think back of the peculiar request He Yi made to you: bring back non-deadly weapons. After thinking a while about it, you remember tales about the Pug having special weapons that could easily knock people out. Maybe you should go to <destination> and speak to the Oathkeepers about this.`
+				accept
+	to fail
+		has "Unfettered: Close Quarter: done"
+	on complete
+		conversation
+			`You cannot find Admiral Danforth, and wager he's probably busy with the pirates above. You think of disturbing him, but another Oathbreaker sees you and asks what you're looking for. You start speaking with him of the Pug war, and recognizing you, he seems eager to share his experience.`
+			`	"I was a simple soldier fighting with the rest of the fleet, in the Republic, at that time. We managed to find an isolated and disabled Pug ship. That by itself seemed like an opportunity too good to let go. Given we were in a Cruiser, we did not fear the crew of such a small ship. But we were wrong: as soon as we boarded said ship, we knew something was wrong. But we were committed, so we went in anyway. Being a soldier, I was on the frontline, and I saw things... Things you do not want to see, trust me. I saw people falling for no apparent reason. But that was not the worst of it. As you can see, I have survived, and that by itself is terrifying. They shot me! They shot me, and yet I lived."`
+			`	You thank him for telling you this story and then think to yourself. This means there is a good chance the Pug have the non-lethal weapons you are looking for. Now you just have to find the Pug, but they'll probably not give you these weapons willingly...`
+
+
+
+mission "Unfettered: Know Your Enemy"
+	name `Acquire a translator`
+	description `You should use your pseudo-independence from the Unfettered to scan Wanderer space. Also, you need to obtain a translator. First go meet the Wanderers, and then go to <stopovers> to see if He Yi managed to find one.`
+	source "Firelode"
+	stopover "Warfeed"
+	to offer
+		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
+	on offer
+		conversation
+			`Moving through the spaceport, you notice some buildings are not as dilapidated as before. You wonder if the Unfettered have found new interest in restoring their past, or if the new Hai arrivals brought this about. Whatever the case is, this does not look like a place of conflict anymore. It looks like a place of hope.`
```

Comment:
Why would we wonder if the Unfettered are interested in restoring their past? I don't think there's been much context to lead us to believe this.

Reply:
I mean, you see the buildings restored, this seems like a reasonable supposition to happen without context?
I mostly meant "you wonder if the UHai put the funds into restoring this or the Hai did"